src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskReplicationMIControllerTest.java

```diff
@@ -113,7 +135,111 @@ public void user_should_configure_task_and_configure_action_recorded_in_replica_
+            .body("task_history_list.get(0).updated", notNullValue())
+            .body("task_history_list.get(0).update_action", equalTo("Configure"))
+            .body("task_history_list.get(1).state", equalTo("UNASSIGNED"))
+            .body("task_history_list.get(1).assignee", equalTo(null))
+            .body("task_history_list.get(1).updated_by", notNullValue())
+            .body("task_history_list.get(1).updated", notNullValue())
+            .body("task_history_list.get(1).update_action", equalTo("Configure"))
+            .body("task_history_list.get(2).state", equalTo("ASSIGNED"))
+            .body("task_history_list.get(2).assignee", notNullValue())
+            .body("task_history_list.get(2).updated_by", notNullValue())
+            .body("task_history_list.get(2).updated", notNullValue())
+            .body("task_history_list.get(2).update_action", equalTo("Claim"));
+
+        Response resultReportable = restApiActions.get(
+            ENDPOINT_BEING_TESTED_REPORTABLE,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        JsonPath claimJsonPathEvaluator = resultReportable.jsonPath();
+
+        assertTrue(OffsetDateTime.parse(configureJsonPathEvaluator.get("reportable_task_list.get(0).created"))
+                       .isEqual(OffsetDateTime.parse(claimJsonPathEvaluator.get("reportable_task_list.get(0).created"))));
+        assertTrue(OffsetDateTime.parse(claimJsonPathEvaluator.get("reportable_task_list.get(0).created"))
+                       .isBefore(OffsetDateTime.parse(claimJsonPathEvaluator.get("reportable_task_list.get(0).updated"))));
+        assertTrue(OffsetDateTime.parse(configureJsonPathEvaluator.get("reportable_task_list.get(0).updated"))
+                       .isBefore(OffsetDateTime.parse(claimJsonPathEvaluator.get("reportable_task_list.get(0).updated"))));
+        assertNotEquals(configureJsonPathEvaluator.get("reportable_task_list.get(0).updated_by").toString(),
+                        claimJsonPathEvaluator.get("reportable_task_list.get(0).updated_by").toString());
+
+        resultReportable.prettyPrint();
+        resultReportable.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("reportable_task_list.size()", equalTo(1))
+            .body("reportable_task_list.get(0).state", equalTo("ASSIGNED"))
+            .body("reportable_task_list.get(0).assignee", notNullValue())
+            .body("reportable_task_list.get(0).updated_by", notNullValue())
+            .body("reportable_task_list.get(0).updated", notNullValue())
+            .body("reportable_task_list.get(0).update_action", equalTo("Claim"))
+            .body("reportable_task_list.get(0).first_assigned_date", notNullValue())
+            .body("reportable_task_list.get(0).first_assigned_date_time", notNullValue())
+            .body("reportable_task_list.get(0).wait_time_days", equalTo(0))
+            .body("reportable_task_list.get(0).wait_time", notNullValue())
```

Comment:
Should check the interval here looks correct too.  E.g. between 0 and 10 mins is fine.

Reply:
👍